doc/iterations/2024.3/shapes/overflow_label_strategy.adoc

```diff
@@ -0,0 +1,61 @@
+= Overflow label strategy
+
+== Problem
+
+In the existing node layouts, the inside label of a node will always be used to determine the minimum size.
+This means that in the case of a long label, the node will necessarily take a size that allows the label to be displayed without any modification.
+In some cases, specifier doesn't want that the label affects the node representation.
```

Comment:
Honestly I'm not sure that this should even be configurable at all. It may just be me but as an end user, I would find it very confusing if some labels behaved differently from others and as a specifier, I would find it very painful to have to configure every label manually.

I would personally put wrap and ellipsis everywhere and let the choice of the end user regarding the size of the elements be respected. If an end user creates a node of 50x50 nicely positioned next to other nodes, I would find it quite odd to break all the layout and move everything because its label would be too large.

But it may be just me...